Scripts/Services/Dungeons/CovetousVoidSpawn/Region.cs

```diff
@@ -68,7 +68,7 @@ public override void OnDeath(Mobile m)
 
                     if (chance > Utility.RandomDouble())
                     {
-                        Mobile mob = DemonKnight.FindRandomPlayer((BaseCreature)m);
+                        Mobile mob = ((BaseCreature)m).RandomPlayerWithLootingRights();
```

Comment:
Possible InvalidCastException?

Reply:
ill check, that was already cast with the previous code I repleaced.

Reply:
Yes:
if (m is CovetousCreature && ((CovetousCreature)m).VoidSpawn)